rfd/0080-piv-private-key.md

```diff
@@ -0,0 +1,392 @@
+
+With the introduction of a hardware private key, the user's key would not be stored on disk in `~/.tsh`. Instead, it would be generated and stored directly on the hardware key, where it can not be exported. Therefore, if an attacker exfiltrates a user's `~/.tsh` folder, the contained certificates would be useless without also having access to the user's hardware key.
+
+So far, just introducing hardware private keys into the login process prevents simple exfiltration attacks. However, an attacker could still potentially steal a user's login session if they hack into the user's computer while the hardware key is connected. To mitigate this risk, we also need to enforce a presence check. For this, we have two options:
+ 1. Enable [per-session MFA](https://goteleport.com/docs/access-controls/guides/per-session-mfa/) alongside hardware private key enforcement, which requires you to pass an MFA check (touch) to start a new Teleport Service session (SSH/Kube/etc.). 
+ 2. Enforce Touch to access hardware private keys, which can be done with PIV-compatible hardware keys. In this case, Touch is required for every Teleport request, not just new Teleport Service sessions.
+
+The first option is a bit simpler as it rides off the coattails of our existing per-session MFA system. On the other hand, the second option provides better security principles, since touch is enforced for every Teleport request rather than just Session requests, and it requires fewer roundtrips to the Auth server.
+
+In this RFD we'll explore both options together, since they are not mutually exclusive, and the underlying implementation will be the same regardless. 
+
+Note: If either of these options are combined with MFA/PIV PIN enforcement, or biometric key usage (like the [Yubikey Bio Series](https://www.yubico.com/products/yubikey-bio-series/)), then even if a user's computer and hardware key are stolen, the user's login session would not provide access to an attacker. To avoid overcomplicating this RFD, we will omit this consideration and leave it as a possible future improvement.
+
+### Server changes
+
+Hardware private key enforcement will be configured and controlled by the Teleport Auth Server, so let's start with the server changes.
+
+#### Configuration
+
+Teleport admins can require hardware private key storage in the cluster's Auth Preference, which can be defined in the Auth Service's config file or with a dynamic Cluster Auth Preference object:
+
+```yaml
+auth_service:
+  ...
+  authentication:
+    ...
+    private_key_policy: none | hardware_key | hardware_key_touch
```

Comment:
Is this the correct field name? Didn't we decide to roll everything under `require_session_mfa`?

Reply:
Oh I see - below it says that we decided to omit this setting. Can we just remove this section from the RFD then to avoid confusion?